docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,210 @@
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## What's new in this release
+
+PMM 2.43 brings a host of updates, including redesigned dashboards, new collectors and metrics, enhanced troubleshooting capabilities, strengthened security, and expanded platform support with official ARM compatibility for PMM Client.
+
+Additionally, it features many bug fixes and performance enhancements to improve your user experience.
+
+### MongoDB dashboard improvements and GA of recent experimental dashboard
+
+We've made some significant enhancements to MongoDB monitoring, focusing on redesigned dashboards, streamlined navigation, and general availability of previously experimental dashboards.
+
+!!! note alert alert-primary "Note"
+    Experimental dashboards are new or redesigned dashboards initially released as preview features. As experimental dashboards mature and demonstrate their value and stability, they transition to GA status, signifying they are now fully stable and ready for general use.
+
+#### Redesigned dashboards
+
+We've overhauled two critical MongoDB dashboards to provide more value, especially for complex, multi-environment setups:
+
+- **Replica Set Summary**
+- **Sharded Cluster Summary** (formerly Cluster Summary)
+
+These dashboards now offer enhanced usability, improved insights, and optimized visualizations for complex environments. 
+
+They focus on highlighting potential database issues for faster resolution and provide centralized, actionable insights without clutter.
+
+#### General availability of Experimental dashboards
+
+The redesigned dashboards, along with the following previously experimental dashboards, are now generally available:
+
+- **MongoDB Oplog Details**
+- **MongoDB Collection Overview**
+- **Replica Set Summary**
+- **Sharded Cluster Summary** (formerly Cluster Summary)
+
+#### Accessing the new dashboards
+
+You can find the updated dashboards in the **MongoDB** and **Dashboards** sections of the main menu. They replace the previous versions as the default dashboards for MongoDB monitoring.
+
+The old dashboards have been moved to the **Experimental** folder and renamed with **(Old)** appended to their names. Future releases will remove these older versions.
+
+#### Updated MongoDB menu structure
+
+To complement the dashboard improvements, we've also restructured the MongoDB section on the main menu for better navigation and accessibility:
+
+- Reorganized menu for a more intuitive navigation, prioritizing the two updated GA dashboards above.
+- Decluttered view by moving older dashboards to the **Experimental** folder or removing them from the menu.
+- Added **MongoDB Oplog Details**, **MongoDB Collection Overview**, and **MongoDB Instance Summary** to the main menu.
```

Comment:
The image and the words don't match.  We use the expanded dashboard name in text but the image is a shortened version...not a huge deal but they're out of order so I'm scanning the list to see that Oplog is last (But listed first) and Summary is first (but listed last).